.php_cs.dist

```diff
@@ -1,11 +1,8 @@
 <?php
 
-$year = date('Y');
```

Comment:
shouldn't we retain this behavior to force update the year in 2019?

Reply:
@borNfreee 

Ha, when the time comes, only this needs to be updated:

https://github.com/infection/infection/blob/6b800242c88187b72e7bd6690071933ddb9e0877/LICENSE#L3